DemoApp-Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/SplashActivity.java

```diff
@@ -0,0 +1,32 @@
+package com.applovin.enterprise.apps.demoapp;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.text.Html;
+import android.widget.TextView;
+
+import com.applovin.sdk.AppLovinSdkUtils;
+
+import java.util.concurrent.TimeUnit;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class SplashActivity
+        extends AppCompatActivity
+{
+    @Override
+    protected void onCreate(Bundle savedInstanceState)
+    {
+        super.onCreate( savedInstanceState );
+        setContentView( R.layout.activity_splash );
+
+        TextView logoTextView = findViewById( R.id.logoTextView );
+        logoTextView.setText( Html.fromHtml( "<b>App</b>Lovin" ) );
```

Comment:
Do we need to set this here, looks like we are already setting it in the [layout file](https://github.com/AppLovin/AppLovin-MAX-SDK-Android/blob/2180ff4ac6c7ac1d2c84d31a2c252186e32d50af/DemoApp-Java/app/src/main/res/layout/activity_splash.xml#L27).

Same for Kotlin project.